Sources/UnleashProxyClientSwift/Context.swift

```diff
@@ -1,12 +1,12 @@
 public struct Context {
-    let appName: String?
-    let environment: String?
-    var userId: String?
-    var sessionId: String?
-    var remoteAddress: String?
-    var properties: [String: String]?
+    public let appName: String?
+    public let environment: String?
+    public var userId: String?
+    public var sessionId: String?
+    public var remoteAddress: String?
+    public var properties: [String: String]?
     
-    init(
+    public init(
```

Comment:
Why do we need to change these? Especially the `var` ones that can be modified later on, makes me feel a bit unsafe...

Reply:
Please check the comment from @jlubawy

Reply:
I understand, but if the purpose is to be able to read them, maybe an accessor would be better. I usually try to avoid mutable objects, in particular when used for context. Another alternative would be to provide a clone method for the Context with a properties parameter.

Reply:
@jlubawy we exposed public toMap on the context to avoid mutable fields

Reply:
That's reasonable. I think `public private(set)` or `public internal(set)` could have worked here too if you don't want it to be mutable.